src/components/views/rooms/EventTile.js

```diff
@@ -564,7 +569,8 @@ module.exports = WithMatrixClient(React.createClass({
 }));
 
 module.exports.haveTileForEvent = function(e) {
-    if (e.isRedacted()) return false;
+    // Only messages have a tile (black-rectangle) if redacted
+    if (e.isRedacted() && e.getType() !== 'm.room.message') return false;
```

Comment:
We should probably display tiles for all redacted events. Presumably this won't really require anything extra, as long as we use the same style for now?